### PR TITLE
Evan building after setup

### PR DESCRIPTION
**Pull request by Evan-Kreutzwiser**
Pull the building code to main even though settlements cannot be built, so that new branches can use the road building and cancel button

### CODE REVIEW

File: KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java [hunk 1]
```diff
@@ -200,6 +200,7 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
         buildRoadRbtn.setSelected(true);
         buildRoadRbtn.setText("Road");
 
+        buildBtn.setFont(new java.awt.Font("Dialog", 1, 13)); // NOI18N
```
Comment (Tacitor) [inline KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java:203]: What's with this font?

Comment (Evan-Kreutzwiser) [reply KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java:203]: I made the font the tiniest bit bigger

Comment (Evan-Kreutzwiser) [reply KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java:203]: Should i undo it?

File: KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java [hunk 2]
```diff
@@ -452,17 +476,43 @@ public void mouseClick(MouseEvent event) {
...
                             if (inSetup && playerSetupRoadsLeft > 0) {
                                 roadNodes.get(i).setPlayer(currentPlayer);
-                                buildingObject = 0;
-                                showRoadHitbox = false;
                                 playerSetupRoadsLeft--;
-                                repaint();
 
                             }
+                            // If the real game is in progress and the user has the cards needed
+                            if (findCards(1, 1) && findCards(2, 1)) {
```
Comment (Tacitor) [inline KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java]: Shouldn't this be an else if?

**Review by Tacitor — COMMENTED** [review]
Looks pretty good.

File: KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java (hunk 1 above)
Comment (Tacitor) [reply KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java:203]: No that's fine. I was just talking about the typeface, but from what you say it seems it was always "Dialog"

File: KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java (hunk 2 above)
Comment (Evan-Kreutzwiser) [reply KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java]: yeah it should
I'll change it

Comment (Evan-Kreutzwiser) [reply KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java]: I changed in with the github site

**Review by Tacitor — APPROVED** [review]
Very epic

**Pull request merged**